web/src/lib/components/MetaTag.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+
+    export let title: string = "Plebeian Market";
+    export let description: string = "Self-sovereign marketplace built on Bitcoin and Lightning";
+    export let image: string = "https://plebeian.market/images/logo.jpg";
+    export let domain: string  = "plebeian.market";
+
+    let defaultTitle = "Plebeian Market"
+    let baseTitle: string = " | Plebeian Market"
+    let url: string;
+
+    onMount(() => {
+        url = window.location.href
+        // concat title if not on home page
+        if (title !== defaultTitle) {
+            title = title.concat(baseTitle);
+        }
+    });
+</script>
+
+
+<title>{title}</title>
+
+<!-- HTML Meta Tags -->
+<meta name="description" content={description}>
+
+<!-- Facebook Meta Tags -->
```

Comment:
these are OG tags - they are used by many websites, not just FB. Better remove this comment.